indexing.rdf

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="owl2html.xsl"?>
+<rdf:RDF 
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+
+    <owl:Ontology rdf:about="http://fedora.info/definitions/v4/indexing#">
+        <rdfs:label xml:lang="en">Fedora Commons Indexing Ontology</rdfs:label>
+        <rdfs:comment xml:lang="en">Ontology for user-defined indexing properties and classes on Fedora-curated resources.</rdfs:comment>
+    </owl:Ontology>
+
+    <owl:DatatypeProperty rdf:about="http://fedora.info/definitions/v4/indexing#hasIndexingTransformation">
+        <rdfs:label xml:lang="en">has indexing transformation</rdfs:label>
+        <rdfs:domain rdf:resource="http://fedora.info/definitions/v4/indexing#Indexable"/>
+        <rdfs:range rdf:resource="&xsd;string"/>
```

Comment:
I don't think this is a URI, since it's just the name of the transform not the full URI to it.  So we could restrict this to xsd:NMTOKEN if we wanted to be more restrictive than xsd:string.